### PR TITLE
bali-phy 3.6.0

### PR DESCRIPTION
**Pull request by bredelings**
- [X] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

**Comment by bredelings** [comment]
It looks like builds are actually succeeding, but are being reported as failed because the _dependencies_ haven't switched to the new sha256/cellar syntax yet.

**Comment by sjackman** [comment]
I've merged PR https://github.com/brewsci/homebrew-bio/pull/1274 to fix the style issues. You can now rebase this PR onto `origin/develop`.

**Comment by bredelings** [comment]
Thank you!  Rebased.

File: Formula/bali-phy.rb [hunk 1]
```diff
@@ -24,16 +24,17 @@ class BaliPhy < Formula
...
   depends_on "pkg-config" => :build
   depends_on "cairo"
-  depends_on "gcc@8" unless OS.mac? # for C++17
+  depends_on "gcc@9" unless OS.mac? # for C++17
 
   # C++17
   fails_with gcc: "5"
   fails_with gcc: "6"
   fails_with gcc: "7"
+  fails_with gcc: "8"
```
Comment (sjackman) [inline Formula/bali-phy.rb:33]: Unless it actually fails to build with GCC 8, you don't need to add this line.

**Comment by sjackman** [comment]
Thanks again for your contributions to Brewsci/bio, Benjamin! 🏆

**Pull request merged**